docs/model_developers/project_creation.rst

```diff
@@ -47,7 +47,7 @@ An example ``project.yml`` could be:
   install_mode: release
   requires:
   pip:
-      index-url: https://pypi.org/simple/
+      index-url: https://my_repo.secure.company.com/repository/inmanta-production
```

Comment:
I personally see no reason not to use PyPi here. It is a valid index for simple projects that use the open source modules only. If we really want to avoid it, I think we must use `example.com` because that is a reserved domain while `company.com` is not.

Reply:
I tend to agree and prefer pypi